### PR TITLE
New package: MoonshotAI.Kimi version 1.1.2

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/188133)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Kimi智能助手\app\resources\lib\parfait_crash_handler.exe returned exit code: -1073741515
> Executable C:\Program Files\Kimi智能助手\Kimi智能助手.exe returned exit code: 2

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/MoonshotAI/Kimi/1.1.2/MoonshotAI.Kimi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- InstallerReturnCode: 1654
+  ReturnResponse: systemNotSupported
+- InstallerReturnCode: 3010
+  ReturnResponse: rebootRequiredToFinish
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- ProductCode: '{9644F42C-C70A-373B-8687-9C4FBBF4CD4C}'
+  UpgradeCode: '{57D320D2-7471-4345-8679-07D645B034CB}'
+  InstallerType: wix
+Installers:
```
Comment (stephengillie) [inline manifests/m/MoonshotAI/Kimi/1.1.2/MoonshotAI.Kimi.installer.yaml:57]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Hi @SpecterShell,

Verify:
image
The installer appeared to run normally, but double-clicking the desktop shortcut doesn't seem to have any effect. Is this expected?

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
> Hi @SpecterShell,
> 
> Verify: image The installer appeared to run normally, but double-clicking the desktop shortcut doesn't seem to have any effect. Is this expected?

Appears to be an encoding bug with Java. Both the default installation directory and the name of config file inside contain Chinese characters, so the program would crash at startup if the non-Unicode encoding was not set to Simplified Chinese.

**Comment by stephengillie** [comment]
That's a rather major bug. How should this PR proceed?

**Review by ItzLevvie — APPROVED** [review]

**Comment by SpecterShell** [comment]
> That's a rather major bug. How should this PR proceed?

Simply merging it? Users in need of this package should not be affected by that. 👻

**Review by ItzLevvie — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**